### PR TITLE
[657] improve setup instructions

### LINKED ISSUE #657: Improve set up instructions
> ### Placeholder text / ticket
> As part of the Canarie project, we have identified the need to make the instructions a bit friendlier for newcomers to Arranger.

### PR DESCRIPTION
**Pull request by caravinci**
Adds additional details on how to setup and run the project, hopefully in a friendlier manner from a newcomer perspective.

Closes #657.

### CODE REVIEW

**Review by joneubank — APPROVED** [review]
Thanks for all the little things here.

If you want to be even more OCD... we are inconsistent with the use of `Title Case` or `Sentence case` in our headers... but don't let that distract from the good work here.

**Comment by caravinci** [comment]
Good point @joneubank! Changed the casing, and also corrected my commonwealth spellings (e.g. z vs s, ou vs o, etc.)

File: README.md [hunk 1]
```diff
@@ -11,93 +11,124 @@ Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/bu
...
-npm run bootstrap
+### Getting Started
 
-# watch modules and rebuild on change
-npm run watch
+#### - Development Setup
 
-# run tests for all modules
-npm test
+Setting up the project, and ready things to make changes
```
Comment (b-f-chan) [inline README.md:30]: Minor and optional suggestion "...and prepare things to make changes"

Comment (caravinci) [reply README.md:30]: Oh I like that!

File: README.md [hunk 2]
```diff
@@ -11,93 +11,124 @@ Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/bu
...
 ```
 
-- [Documentation](#Documentation)
-- [Motivation](#motivation)
-- [What is a "Data Portal"?](#data-portal)
-  - [Topology](#topology)
-- [Roadmap](#roadmap)
-- [Development](#development)
-- [License](https://github.com/overture-stack/arranger/blob/master/LICENSE)
+Now you should be able to start the following processes from the project's root:
```
Comment (b-f-chan) [inline README.md:47]: Suggest adding "...from the project's root **folder**:" for extra clarity

Comment (caravinci) [reply README.md:47]:
```suggestion
Now you should be able to start the following processes from the project's root folder:
```

File: README.md [hunk 3]
```diff
@@ -11,93 +11,124 @@ Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/bu
...
 ```bash
-make help
-```
+# Start all services at once, using some default settings.
+# This runs the following services: Elasticsearch, kibana, arranger-server, and arranger-ui
+  make start
+
+# ^^^ which runs the following command behind the scenes:
+# ES_USERNAME=elastic ES_PASSWORD=myelasticpassword docker-compose -f docker-compose.yml up -d -build
+# Note: these ES_* values may be customized when running your own Arranger instance
```
Comment (b-f-chan) [inline README.md:80]: Is it possible to add a small section on how these can be customized/configured post-install?

Comment (caravinci) [reply README.md:80]: They are not meant to be changed easily once the images are built... too many moving parts.

File: README.md [hunk 4]
```diff
@@ -11,93 +11,124 @@ Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/bu
...
-### Documentation
 
-For in-depth documentation, please see the Arranger Read the Docs: https://arranger.readthedocs.io/en/latest/
+---
+# Afterwards, in another bash process, you may seed an example file_centric index
+  make init-es
+
+# ^^^ which runs the following command behind the scenes:
+# ./docker/elasticsearch/load-es-data.sh ./docker/elasticsearch elastic myelasticpassword
+# That SH script may give you ideas on how to automate uploading indexes to your instance.
```
Comment (b-f-chan) [inline README.md:89]: Just to clarify, this is additional enhanced scripting they can do themselves correct?  But not necessary/mandatory for base install?  (i.e. I don't they don't necessarily need to write a script to automate correct?  i.e. they don't need to write their own Maestro :) )

File: README.md [hunk 5]
```diff
@@ -11,93 +11,124 @@ Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/bu
...
+
+
+---
+# Bonus: ----------------------------- #
+# See other preprogrammed make targets
+  make help
+# e.g. utilities to list the indexes, or clear the Elasticsearch; list the running docker containers, etc.
```
Comment (b-f-chan) [inline README.md:96]: Are these utilities available for users from within Arranger itself?  i.e. Should I include a reference/list of what all these utilities are?

File: README.md (hunk 4 above)
Comment (caravinci) [reply README.md:89]: That's correct. This was just a pointer, for those who want to seed a different set of scaffoldings, or are not using maestro at all.

File: README.md [hunk 6]
```diff
@@ -131,11 +162,13 @@ _this is way too simplistic. needs an update_
...
+
+### Development Details
 
 Arranger is a [lerna](https://github.com/lerna/lerna) flavored [monorepo](https://medium.com/@maoberlehner/monorepos-in-the-wild-33c6eb246cb9). The modules exposed by Arranger compose all of the necessary code required to build an application such as the [Genomic Data Commons](https://portal.gdc.cancer.gov/).\*
 
-### Releases
+#### Releasing Instructions
 
 - From `master` branch, run `npm run tag <version>`
 - Publishing process will be run [by Jenkins](https://jenkins.qa.cancercollaboratory.org/blue/organizations/jenkins/Overture.bio%2Farranger/activity?branch=master)
```
Comment (b-f-chan) [inline README.md:174]: Is this applicable to the user?  It points to our internal Jenkins.  What would the user have to do themselves to publish?  Have their own Jenkins pipeline?  Is this necessary for them?

File: README.md (hunk 5 above)
Comment (caravinci) [reply README.md:96]: These are development tools, useful only outside the Arranger containers.
I don't think adding the references would be useful to them.

File: README.md (hunk 6 above)
Comment (caravinci) [reply README.md:174]: This is something the docs already had, to indicate to developers how to release a new version.

While making all these changes, I was thinking this better belongs in a standalone contribution guide, but it's TODO for the future.

**Pull request merged**